package.json

```diff
@@ -24,6 +24,7 @@
     "test:consumer": "yarn workspace consumer-test test",
     "test:devops": "npm-run-all test:teams test:mocha:junit test:nyc:cobertura",
     "test:github": "npm-run-all test:mocha:min test:runtime:min test:teams:min test:nyc:lcov",
+    "test:orchestrator": "wsrun -m -p \"botbuilder-ai-orchestrator\" -t test",
```

Comment:
The tests should already be picked up in `test:github` and `test:devops`, are they not?

Reply:
We will use this when we build from our build system to validate integrations from core..  Just to isolate to our specific tests.

Reply:
Ok - since you're targeting a single workspace this should be `"test:orchestrator": "yarn workspace botbuilder-ai-orchestrator test"` then. The `wsrun` tool is intended to run against several packages.

Reply:
Got it - fixed.